.github/workflows/npm-auto-publish-alpha.yaml

```diff
@@ -0,0 +1,65 @@
+# This Github Action is designed to automatically publish NPM packages.
+#
+# While this Github Action will often be triggered, it will only publish a new package
+# if the version specified in the package.json file(s) is behind that already published.
+#
+# Further documentation on how this webhook works can be found at the following URL:
+# https://github.com/marketplace/actions/npm-publish
+
+name: "NPM Auto Publish: Alpha"
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - "solana/gatekeeper-lib/package.json"
```

Comment:
Everything that is not under "program_v2" is "v1" and I think we should leave it out of the automation for now.